docker_scripts/squash.py

```diff
@@ -518,8 +520,8 @@ def run(self):
             # without loading into Docker
             self._tar_image(self.output_path, new_image_dir)
             self.log.info("Image available at '%s'" % self.output_path)
-        else:
-            # Load image into Docker
+        if not self.output_path or self.load_output_back:
```

Comment:
Ok, I don't have this feeling but I understand that you may have it. What would work better for you? `self.load_output_back or not self.output_path`? Something different? I have no problem with changing this.